core/templates/pages/exploration-editor-page/translation-tab/state-translation/state-translation.component.spec.ts

```diff
@@ -0,0 +1,922 @@
+      explorationStatesService = $injector.get('ExplorationStatesService');
+      routerService = $injector.get('RouterService');
+      translationLanguageService = $injector.get('TranslationLanguageService');
+      translationTabActiveContentIdService = $injector.get(
+        'TranslationTabActiveContentIdService');
+      translationTabActiveModeService = $injector.get(
+        'TranslationTabActiveModeService');
+
+      spyOn(stateEditorService, 'getActiveStateName').and.returnValue(
+        'Introduction');
+      explorationStatesService.init(explorationState3);
+      stateRecordedVoiceoversService.init('Introduction',
+        recordedVoiceoversObjectFactory.createFromBackendDict(
+          recordedVoiceovers));
+
+      $scope = $rootScope.$new();
+      ctrl = $componentController('stateTranslation', {
+        $rootScope: $rootScope,
+        $scope: $scope,
+        CkEditorCopyContentService: ckEditorCopyContentService,
+        StateEditorService: stateEditorService
+      }, {
+        isTranslationTabBusy: false
+      });
+      ctrl.$onInit();
+    }));
+
+    it('should evaluate hint tab as enabled', function() {
+      expect($scope.isDisabled('content')).toBe(false);
+    });
+
+
+    it('should evaluate feedback tab as disabled', function() {
+      expect($scope.isDisabled('feedback')).toBe(true);
+    });
+
+    it('should evaluate hint tab as disabled', function() {
+      expect($scope.isDisabled('hint')).toBe(true);
+    });
+
+
+    it('should evaluate solution tab as disabled', function() {
+      expect($scope.isDisabled('solution')).toBe(true);
+    });
+  });
+
+  describe('when state has no hints, solution and outcome', function() {
+    beforeEach(angular.mock.inject(function($injector, $componentController) {
+      $rootScope = $injector.get('$rootScope');
+      explorationStatesService = $injector.get('ExplorationStatesService');
+      routerService = $injector.get('RouterService');
+      translationLanguageService = $injector.get('TranslationLanguageService');
+      translationTabActiveContentIdService = $injector.get(
+        'TranslationTabActiveContentIdService');
+      translationTabActiveModeService = $injector.get(
+        'TranslationTabActiveModeService');
+
+      spyOn(stateEditorService, 'getActiveStateName').and.returnValue(
+        'Introduction');
+      explorationStatesService.init(explorationState4);
+      stateRecordedVoiceoversService.init('Introduction',
+        recordedVoiceoversObjectFactory.createFromBackendDict(
+          recordedVoiceovers));
+
+      $scope = $rootScope.$new();
+      ctrl = $componentController('stateTranslation', {
+        $rootScope: $rootScope,
+        $scope: $scope,
+        CkEditorCopyContentService: ckEditorCopyContentService,
+        StateEditorService: stateEditorService
+      }, {
+        isTranslationTabBusy: false
+      });
+      ctrl.$onInit();
+    }));
+
+    it('should evaluate feedback tab as disable', function() {
```

Comment:
```suggestion
    it('should evaluate feedback tab as disabled', function() {
```

Ditto below: disable -> disabled

Reply:
Done.